tests/test_modeling_common.py

```diff
@@ -1747,12 +1747,13 @@ def test_resize_position_vector_embeddings(self):
             self.assertTrue(models_equal)
 
     def test_resize_tokens_embeddings(self):
+        if not self.test_resize_embeddings:
+            self.skipTest(reason="test_resize_embeddings is set to `False`")
```

Comment:
(moved the skip here: no point in spending compute if we are going to skip the test)